src/python/visclaw/frametools.py

```diff
@@ -186,6 +186,21 @@ def plot_frame(framesolns,plotdata,frameno=0,verbose=False):
             beforeaxes = getattr(plotaxes,'beforeaxes',None)
             current_data = run_str_or_func(beforeaxes,current_data)
                 
+            skip_patches_outside_xylimits = plotaxes.skip_patches_outside_xylimits
+            
+            if skip_patches_outside_xylimits is None:
+                # User didn't set.  Set to True unless there's a mapped grid
+                
+                MappedGridExists = (plotdata.mapc2p is not None)
```

Comment:
Just a minor point for Python style.  Variables that are not classes are usually lower case and underscored so that `MappedGridExists` should be `mapped_grid_exists`.

Reply:
Good point, I changed it to `mapc2p_exists` as more descriptive.

I chose the original name because there is also a `plotitem` attribute named `MappedGrid`.   But I don't check that in the test done in this PR since it seemed simpler and more cautious to just look for a `mapc2p` attribute even if `MappedGrid` is `False` (An option allowed in case the user wants to see things in the computational plane even if there is a grid mapping.)

Reply:
See name change in #280.